apps/android_rpc/app/src/main/jni/Android.mk

```diff
@@ -38,7 +38,8 @@ LOCAL_LDFLAGS := -L$(SYSROOT)/usr/lib/ -llog
 
 LOCAL_C_INCLUDES := $(ROOT_PATH)/include \
                     $(ROOT_PATH)/3rdparty/dlpack/include \
-                    $(ROOT_PATH)/3rdparty/dmlc-core/include
+                    $(ROOT_PATH)/3rdparty/dmlc-core/include \
+                    $(ROOT_PATH)/3rdparty/compiler-rt
```

Comment:
compiler-rt should be directly included using relative path, @islavutin can you point out the file that did not do that?

Reply:
I meant that 3rdparty/compiler-rt does not need to be part of include path, because we suppose to include the files in it via relative ones, e.g.  https://github.com/apache/tvm/blob/main/src/runtime/contrib/random/mt_random_engine.cc#L32

Reply:
so a fix should be change other include of the compiler rt file to relative include. We did not intend to use that particular file as a public header

Reply:
Does that still hold here as this is in the `apps` folder? I'd assume the app is setup so it could be plucked out of the tree?

Reply:
i think it still holds, because the all in one packed still requires the src tree due to the mechanism we used to build the packed runtime

Reply:
Hi @tqchen, I have updated the fix based on your recommendations